app/src/main/java/org/oppia/android/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -29,7 +29,8 @@ import javax.inject.Inject
 class DragDropSortInteractionView @JvmOverloads constructor(
   context: Context,
   attrs: AttributeSet? = null,
-  defStyleAttr: Int = 0
+  defStyleAttr: Int = 0,
+  private val singleTypeBuilderFactory: BindableAdapter.SingleTypeBuilder.Factory
 ) : RecyclerView(context, attrs, defStyleAttr) {
```

Comment:
Here we don't have @Inject notation.

Reply:
You'll need to use field injection for custom views. See the ``@Inject`` fields below.

Reply:
I have a doubt here, `SingleTypeBuilder.Factory` depends on `fragment` but we don't have a fragment dependency here, so will it work?

Reply:
Yup, it should (assuming the view is parented by a fragment which most in Oppia Android are). Our Dagger hierarchy is set up such that the fragment component inherits from activities, and views inherit from fragments. That means we can have view-only bindings, and views have access to all other bindings in the codebase (include fragment ones).